src/coreclr/jit/hwintrinsiclistarm64sve.h

```diff
@@ -112,6 +112,7 @@ HARDWARE_INTRINSIC(Sve,           SignExtend32,
 HARDWARE_INTRINSIC(Sve,           SignExtend8,                                                      -1,      -1,     false, {INS_invalid,        INS_invalid,        INS_sve_sxtb,       INS_invalid,        INS_sve_sxtb,       INS_invalid,        INS_sve_sxtb,       INS_invalid,        INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_EmbeddedMaskedOperation|HW_Flag_LowMaskedOperation)
 HARDWARE_INTRINSIC(Sve,           SignExtendWideningLower,                                          -1,      1,      true,  {INS_sve_sunpklo,    INS_invalid,        INS_sve_sunpklo,    INS_invalid,        INS_sve_sunpklo,    INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_BaseTypeFromFirstArg)
 HARDWARE_INTRINSIC(Sve,           SignExtendWideningUpper,                                          -1,      1,      true,  {INS_sve_sunpkhi,    INS_invalid,        INS_sve_sunpkhi,    INS_invalid,        INS_sve_sunpkhi,    INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_BaseTypeFromFirstArg)
+HARDWARE_INTRINSIC(Sve,           Store,                                                            -1,      3,      true,  {INS_sve_st1b,       INS_sve_st1b,       INS_sve_st1h,       INS_sve_st1h,       INS_sve_st1w,       INS_sve_st1w,       INS_sve_st1d,       INS_sve_st1d,       INS_sve_st1w,       INS_sve_st1d},    HW_Category_MemoryStore,           HW_Flag_Scalable|HW_Flag_BaseTypeFromFirstArg|HW_Flag_ExplicitMaskedOperation|HW_Flag_SpecialImport|HW_Flag_SpecialCodeGen|HW_Flag_LowMaskedOperation)
```

Comment:
Did you check if `HW_Flag_BaseTypeFromFirstArg` is needed?

Reply:
We need the flag as stores return `void`, the `simdBaseJitType` doesn't get populated and we hit the assert failure in [impHwIntrinsic()](https://github.com/dotnet/runtime/blob/main/src/coreclr/jit/hwintrinsic.cpp#L1244)